src/store/mutations.js

```diff
@@ -45,13 +45,22 @@ export default {
       filterValues.push(item.value)
       filterTexts.push(item.text)
     }
-
-    Vue.set(state.filters.selections, name, [...new Set(filterValues)])
-    Vue.set(state.filters.labels, name, [...new Set(filterTexts)])
+    if (filterValues.length !== 0) {
+      Vue.set(state.filters.selections, name, [...new Set(filterValues)])
+      Vue.set(state.filters.labels, name, [...new Set(filterTexts)])
+    } else {
+      Vue.delete(state.filters.selections, name)
+      Vue.delete(state.filters.labels, name)
```

Comment:
don't delete labels, they are a caching mechanism

Reply:
Resolved in 861684c